static/docs/user-guide/autocomplete.md

```diff
@@ -73,6 +73,36 @@ fi
 You can `source` your `~/.bash_profile` or launch a new terminal to utilize
 completion.
 
+<details>
+
+### Click to expand if it doesn't work on Debian/Ubuntu
+
+As mentioned above, it should work out of the box. But if it does not, try these steps:
+
+- Make sure that the package `bash-completion` is installed:
+  ```bash
```

Comment:
put extra space before and after code blocks

Reply:
Also it should be ```dvc please, not `bash`. It's a peculiarity of our docs engine.

Reply:
yep, only for cli. And use `$ cmd` syntax